src/main/java/racingcar/view/ResultView.java

```diff
@@ -1,24 +1,46 @@
 package racingcar.view;
 
 import racingcar.model.Car;
+import racingcar.model.Garage;
 
 import java.util.List;
 
 public class ResultView {
 
     private static final String MOVE_UNIT = "-";
 
-    public static void print(List<Car> cars){
-        for (Car car : cars) {
+    public static void print(Garage garage) {
+        for (Car car : garage.getCars()) {
             printCarMovement(car);
         }
         System.out.println();
     }
 
     private static void printCarMovement(Car car) {
+        System.out.print(car.getName() + " : ");
         for (int i = 0; i < car.getMoveCount(); i++) {
             System.out.print(MOVE_UNIT);
         }
         System.out.println();
     }
+
+    public static void printWinner(Garage garage) {
```

Comment:
우승자를 출력하기 위한 메서드이니 우승자를 인자로 받으면 어떨까요?
```suggestion
    public static void printWinner(List<String> winners) {
```